ecs-cli/modules/cli/compose/adapter/convert.go

```diff
@@ -222,6 +222,20 @@ func ConvertToTimeInSeconds(d *time.Duration) *int64 {
 	return &val
 }
 
+func ConvertDurationStrToSeconds(d string) (*int64, error) {
```

Comment:
Since this is a public function, can we add a comment to explain what it does?

> Every exported (capitalized) name in a program should have a doc comment. 

https://golang.org/doc/effective_go.html#commentary